web/lib/firebase/contracts.ts

```diff
@@ -42,17 +42,19 @@ export function contractUrl(contract: Contract) {
 }
 
 export function contractMetrics(contract: Contract) {
-  const { createdTime, resolutionTime, isResolved } = contract
+  const { createdTime, resolutionTime, isResolved, autoResolutionTime } =
+    contract
 
   const createdDate = dayjs(createdTime).format('MMM D')
+  const autoResolutionDate = dayjs(autoResolutionTime).format('MMM D')
```

Comment:
Ideally, this would display the year if the year is not the same as our current year.

I'd either add that logic, or remove, since nothing is using the autoResolutionDate at the moment.